client/src/app/(main)/message/company/[conversationId]/page.tsx

```diff
@@ -15,7 +15,8 @@ export default function Page({
       }}
     >
       <Conversation conversationId={conversationId} />
-      <OpportunityStats />
+      {/* TODO: NEED ACTUAL A WAY TO PASS IN THE opportunity_id */}
+      <OpportunityStats opportunity_id={opportunity_id} />
```

Comment:
you can just pass in a random id for this

Reply:
ok